glotaran/builtin/models/kinetic_spectrum/test/test_kinetic_spectrum_model.py

```diff
@@ -1,382 +1,233 @@
 import numpy as np
 import pytest
 
+from glotaran import read_model_from_yaml
+from glotaran import read_parameters_from_yaml
 from glotaran.analysis.optimize import optimize
 from glotaran.analysis.scheme import Scheme
-from glotaran.builtin.models.kinetic_spectrum import KineticSpectrumModel
-from glotaran.parameter import ParameterGroup
 
+MODEL_1C_BASE = """\
```

Comment:
For the naming pattern to be consistent with the rest of the code the `1C` should be spelled out, e.g. `MODEL_1C_BASE` should be `MODEL_1_COMPONENT_BASE`.

Reply:
Although in general I would agree with increased verbosity, in particular in any function or class names, 
this is only a constant used in supporting classes of unit tests. 

After careful consideration (trying different things for a few hours while refactoring) I found this to be the right balance between verbosity, readability and maintainability. Maybe in the future we find a better way. I'll be sure to smash my fist on the table when I do.